charts/actions-runner-controller/values.yaml

```diff
@@ -39,6 +39,19 @@ image:
   actionsRunnerRepositoryAndTag: "summerwind/actions-runner:latest"
   dindSidecarRepositoryAndTag: "docker:dind"
   pullPolicy: IfNotPresent
+  # Defines secret to use as ImagePullSecrets in the runner pod spec
+  # actionsRunnerPullImageSecretName: someSecretName
+
+
+imagePullSecrets:
+- name: github-actions-docker-config
```

Comment:
Apparently, this is used from nowhere?

```suggestion
```

Reply:
Well, it turns out this already exists in L57 and is used to set image pull secrets for controller-manager, not runners 😄